pkg/kobject/kobject.go

```diff
@@ -32,30 +32,28 @@ type KomposeObject struct {
 
 // ConvertOptions holds all options that controls transformation process
 type ConvertOptions struct {
-	ToStdout                    bool
-	CreateD                     bool
-	CreateRC                    bool
-	CreateDS                    bool
-	CreateDeploymentConfig      bool
-	BuildRepo                   string
-	BuildBranch                 string
-	Build                       string
-	CreateChart                 bool
-	GenerateYaml                bool
-	GenerateJSON                bool
-	EmptyVols                   bool
-	InsecureRepository          bool
-	Replicas                    int
-	InputFiles                  []string
-	OutFile                     string
-	Provider                    string
-	Namespace                   string
-	IsDeploymentFlag            bool
-	IsDaemonSetFlag             bool
-	IsReplicationControllerFlag bool
-	IsReplicaSetFlag            bool
-	IsDeploymentConfigFlag      bool
-	IsNamespaceFlag             bool
+	ToStdout               bool
+	CreateDeploymentConfig bool
+	BuildRepo              string
+	BuildBranch            string
+	Build                  string
+	CreateChart            bool
+	GenerateYaml           bool
+	GenerateJSON           bool
+	EmptyVols              bool
+	InsecureRepository     bool
+	Replicas               int
+	InputFiles             []string
+	OutFile                string
+	Provider               string
+	Namespace              string
+	Controller             string
+	IsDeploymentFlag       bool
+	//IsDaemonSetFlag             bool
+	//IsReplicationControllerFlag bool
```

Comment:
Why commented out? We are still going to be using these for the deprecated options.